real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/DataPanel.java

```diff
@@ -136,6 +137,10 @@ public void actionPerformed(String topicName, Message m) {
             }
         });
 
+        new Subscriber("compass", NodeChannel.IMU_COMPASS.getMsgPath(), ((topicName, m) -> {
```

Comment:
It doesn't look like we're doing anything with this subscriber in the DataPanel class.  Is this something for another time?